src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -691,6 +697,17 @@ void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
     }
 }
 
+void GraphOptimizer::RemoveConvertF16(Graph& graph) {
+    auto& graphNodes = graph.GetNodes();
+
+    for (auto parent : graphNodes) {
+        if (parent->getType() == Type::Convert && parent->isConstant() && parent->getOriginalInputPrecisionAtPort(0) == Precision::FP16
```

Comment:
Lets also check that child node is FC to avoid unpredictable behavior.

Reply:
done